tools/Makefile

```diff
@@ -136,7 +135,7 @@ $(foreach tool, $(tools-y), $(if $(wildcard $(curdir)/$(tool)/patches),$(eval $(
 $(foreach tool, $(filter-out xz,$(tools-y)), $(eval $(curdir)/$(tool)/compile += $(curdir)/xz/compile))
 
 # make any tool depend on tar, xz and patch to ensure that archives can be unpacked and patched properly
-tools-core := tar xz patch
+tools-core := tar libdeflate xz patch
```

Comment:
@neheb think a comment got missed here...
My suggestion was to split also this on one tool per line and just add the new tool... What do you think?